aptos-move/block-executor/src/executor.rs

```diff
@@ -1278,6 +1281,17 @@ where
                                 output.get_write_summary(),
                             )
                         });
+
+                    if !found_module_rw_conflict
```

Comment:
I don't think you need both found_module_rw_conflict and module_rw_conflict. 

you can just here use `block_limit_processor.has_seen_module_rw_conflict() `